src/actions/application.js

```diff
@@ -39,6 +39,37 @@ function* loadMenu() {
   }
 }
 
-export default function* rootSaga() {
-  yield takeLatest(MENU_REQUESTED, loadMenu);
+export const ROLES_REQUESTED = 'ROLES_REQUESTED';
```

Comment:
Given that this is more pages specific, what about moving these to a new file, like ```roles.js``` and ```permissions.js```?